LargeNetworkAnalysisTools.pyt

```diff
@@ -0,0 +1,377 @@
+############################################################################

```

Comment:
@mmorang 
I am not sure what is the motivation behind moving the toolbox from .tbx to .pyt. One downside I see is using the GP tool from the .pyt is missing the help for all the parameters. In a .pyt the help is stored in the .pyt.xml file which you are ignoring from the repo.

Reply:
Pyt makes it easier to have all the parameters defined in one place, see changes when syncing the repo, and have validation in the same place.

I completely forgot about the side panel help, though! Thanks for catching that. I have added it now.